datastream-to-influxdb.py

```diff
@@ -57,68 +57,123 @@ def get_log():
     logging.basicConfig(
         format="%(message)s",
         stream=sys.stdout,
-        level=log_level,
+        level=logging.WARN,
```

Comment:
burde ikke denne da settes på linje 55 i stedet?

Reply:
This is to avoid other modules from logging, for instance the akamai edgeclient will log sensitive information if this is set to debug